src/urllib3/connectionpool.py

```diff
@@ -755,7 +756,10 @@ def urlopen(  # type: ignore[override]
             # Everything went great!
             clean_exit = True
 
-        except EmptyPoolError:
+        except (
+            EmptyPoolError,
+            SSLImportError,
```

Comment:
@pquentin If we use `ImportError` here, it might accidentally catch other import errors and may not be obvious what we are doing here.

Reply:
Why would we need to catch the `ImportError` here? It should bubble up if it's not in the `except` statement?

Reply:
You are right. I can change it to raise `ImportError` here.